### PR TITLE
Religions renamed from endonyms

### PR DESCRIPTION
**Pull request by Methaneus**
#1

### CODE REVIEW

File: CK3Plus/common/scripted_triggers/99_CK3Plus_title_triggers.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	can_create_title_trigger = yes
+}
+
+can_create_kingdom_trigger = {
+	can_create_title_trigger = yes
+}
+
+can_create_empire_trigger = {
+	can_create_title_trigger = yes
+}
```
Comment (schwarherz) [inline CK3Plus/common/scripted_triggers/99_CK3Plus_title_triggers.txt]: I'm not sure how this file got here from the landed_titles branch? I suggest removing it before a merge.

**Review by schwarherz — CHANGES_REQUESTED** [review]
There seem to be some excess files in the commit that I recommend removing before merging.

**Review by schwarherz — APPROVED** [review]
I dropped the extra commit. It's good to go now. Leaving it for @Methaneus to merge.

**Comment by Methaneus** [comment]
> There seem to be some excess files in the commit that I recommend removing before merging.
Probably accidentally merged the wrong branch into it.

**Pull request merged**